pystiche_papers/utils/misc.py

```diff
@@ -25,6 +25,7 @@
     "same_size_output_padding",
     "is_valid_padding",
     "batch_up_image",
+    "join_channelwise",
```

Comment:
If we leave the `AddNoiseChannels` in the paper implementation, this should stay there as well.